### ISSUE TITLE
\u2013 character maps to undefined

### ISSUE THREAD

**Issue by willzone1** [Miserlou/SoundScrape#87]
soundscrape started crashing on me at the same point in a user's like list. I found the culprit to be one song: https://soundcloud.com/emiljoacmusic/emiljo-ac-go-hbd-j-dilla-february-7-1974-february-10-2006
The error I saw was a \i2013 unicode character undefined. This character should correspond to a special dash character that I suspect is used in the soundcloud song title "Emiljo A.C. - Go! - HBD J Dilla - (February 7, 1974 – February 10, 2006)".

**Comment by pydo** [comment]
Could you post the full traceback & the exact command you used? I didn't have any issue downloading the song on windows & linux.

**Comment by Miserlou** [comment]
Works for me (great track too)..

**Comment by Miserlou** [comment]
You should pip install soundscrape --upgrade to get the latest version.

**Comment by willzone1** [comment]
Hi all,
Thanks for your help with this issue. Just knowing that the issue was on my end helped! In the end, the issue seems to have been something to do with passing the song titles thru to the console. In case anyone else has this issue in the future, here's what I did:
1. I'm now calling soundscrape direct from a batch file rather than via an os.system python call.
2. I switched over to Python 2 (which meant a fresh pip install). Previous fresh installs (and @Miserlou 's suggestion of pip --upgrade) had failed to solve the issue, so I'm betting there's something wrong with the way my Windows 7 system was handling special unicode characters in the console with python 3.

Anyway, thanks, and sorry this reply was so long coming (got busy!)
